tests/generators/kzg_4844/main.py

```diff
@@ -208,12 +228,29 @@ def case03_verify_kzg_proof():
         }
 
     # Edge case: Invalid y
-    blob, z = VALID_BLOBS[1], VALID_ZS[1]
-    proof = spec.compute_kzg_proof(blob, z)
+    for y in INVALID_ZS:
+        blob, z = VALID_BLOBS[4], VALID_ZS[1]
```

Comment:
I made this change (testing all of the invalid field element values) because I wanted to test invalid length field elements for the `y` field too. Previously, this was only testing one invalid `y` value.